assets/general/bitcoin/info.json

```diff
@@ -5,7 +5,7 @@
   "explorer": "https://explorer.btc.com",
   "explorerTx": "https://explorer.btc.com/btc/transaction/${ID}",
   "explorerAddress": "https://explorer.btc.com/btc/address/${ID}",
-  "regexAddress": "/^(bc1|[13])[a-km-zA-HJ-NP-Z02-9]{25,39}$/",
+  "regexAddress": "^(bc1|[13])[a-km-zA-HJ-NP-Z02-9]{25,39}$",
```

Comment:
Bech32 Address Format
It comprises 42-62 characters. This format was introduced with the native SegWit.

So I suggest to make max length 62

Reply:
P2PKH addresses (base58, 2009, 26–34 chars)

1..

1EQuRM9nKdzUFUadUXKcaHQCeMR5eGsWwP

P2SH addresses (base58, 2012, 34 chars)

3..

3N5Cyor23hD2LnofPrVQfx5uh2j85Pv2gi

P2WPKH (bech32, 2017, 42 chars)

SegWit

bc1q..

bc1qe8y4708t8333vt3alfdzmf2gfwnjxsmt3htw2x

P2WSH (bech32, 2017, 62 chars)

SegWit

bc1q..

bc1qeklep85ntjz4605drds6aww9u0qr46qzrv5xswd35uhjuj8ahfcqgf6hak

P2TR (bech32m, 2021, 62 chars)

Taproot

bc1p..

bc1pxeg30dfzmnv8pjeurk8fqlmt33u9wtgjl6fjxrqth6hr7hekrh2qtzmhdj